commands/namespace.go

```diff
@@ -86,25 +86,28 @@ var namespaceGetCmd = &cobra.Command{
             }
         }
 
-        namespace, _, err := Client.Namespaces.Get(qualifiedName.GetNamespace())
-
-        if err != nil {
-            whisk.Debug(whisk.DbgError, "Client.Namespaces.Get(%s) error: %s\n", getClientNamespace(), err)
-            errStr := wski18n.T("Unable to obtain the list of entities for namespace '{{.namespace}}': {{.err}}",
-                    map[string]interface{}{"namespace": getClientNamespace(), "err": err})
+        errHandler := func(err error, kind string) error {
```

Comment:
Good idea! Maybe it's a good idea to put it in shared.go so other methods can utilize it. Plus it will serve as a template for debug/error handling to reduce code duplication in similar circumstances.